tests/validators/test_int.py

```diff
@@ -59,8 +60,25 @@ def test_int_py_and_json(py_and_json: PyAndJson, input_value, expected):
         (Decimal('1'), 1),
         (Decimal('1.0'), 1),
         (i64_max, i64_max),
+        (str(i64_max), i64_max),
+        (
+            str(i64_max + 1),
+            Err(
+                "Input should be a valid integer, unable to parse string as an integer "
```

Comment:
this seems unfortunate to me as I suspect that eventually someone will use an int type for a big serial-number-like thing, and I think if we said "unable to parse string as a 64-bit integer" then they would understand why it failed sooner than just saying `1234678910111213141516` isn't an integer.

I'm okay leaving as is for now, it would be nice if it gave a reason for why it failed. I guess having the ability to deserialize into a BigInt would be best, we'll see if this needs to be prioritized based on beta feedback hopefully.